### PR TITLE
Add slack notify on lock status every 24 hours.

### PR DESCRIPTION
**Pull request by nishkrishnan**
TODO:

- [x] Need to do some math to figure out when to actually fire the timer, it should be every 24 hours at a certain time.  Will do that as a followup commit to the PR.

### CODE REVIEW

**Review by samrabelachew — COMMENTED** [review]
Can we implement this in a way that makes using the slack activity a plugable feature of atlantis, similar to how the sns notifier is optional?

File: server/neptune/workflows/activities/main.go [hunk 1]
```diff
@@ -59,6 +60,7 @@ func NewDeploy(deploymentStoreCfg valid.StoreConfig) (*Deploy, error) {
 		dbActivities: &dbActivities{
 			DeploymentInfoStore: deploymentStore,
 		},
+		slackActivities: &slackActivities{},
```
Comment (samrabelachew) [inline server/neptune/workflows/activities/main.go:63]: need to pass in the slack client here I believe

File: server/neptune/workflows/internal/temporal/duration.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"go.temporal.io/sdk/workflow"
+)
+
+type NextDay func(time.Time) time.Time
+
+func NextBusinessDay(d time.Time) time.Time {
+	d = d.Add(24 * time.Hour)
+
+	for d.Weekday() == time.Saturday || d.Weekday() == time.Sunday {
+		d = d.Add(24 * time.Hour)
```
Comment (samrabelachew) [inline server/neptune/workflows/internal/temporal/duration.go:15]: should you add 2 days for saturday?

Comment (nishkrishnan) [reply server/neptune/workflows/internal/temporal/duration.go:15]: This is a for loop, it'll add days until monday

Comment (samrabelachew) [reply server/neptune/workflows/internal/temporal/duration.go:15]: oops missed the loop! lgtm

**Review by samrabelachew — APPROVED** [review]

**Review by samrabelachew — APPROVED** [review]

**Pull request merged**